lib/src/config_handler.dart

```diff
@@ -79,11 +79,18 @@ class ConfigHandler {
     disableTelemetryRegex.allMatches(configString).forEach((element) {
       // Conditional for recording telemetry as being disabled
       if (element.group(1) != ';' && element.group(2) == '0') {
-        telemetryEnabled = false;
+        _telemetryEnabled = false;
       }
     });
   }
 
+  // TODO: determine if we should read from the file every time we
```

Comment:
what about if we store in memory the last time the config file was written to, and then check that each time this getter is called?

Reply:
Closing here and adding a new comment